_posts/spaceranger-23/2020-09-09-logical-turtle-post-spaceranger-23.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: post
+author: spaceranger-23
+title: "spaceranger Logical Turtle Post"
+---
+
+Attach code
+
```

Comment:
let me know when you attach embed code

Reply:
Hey, sorry for the delay. Just attached to the code.